mypy.ini

```diff
@@ -18,6 +19,9 @@ warn_unused_ignores=True
 warn_return_any=True
 no_implicit_reexport=True
 
+; Set this to skip until more of the codebase is typed
+follow_imports = skip
```

Comment:
Since most of the codebase is untyped, following imports results in a lot of useless errors. Maybe when we switch mypy.ini to be on full modules instead of random files we can remove this.